### PR TITLE
[CUMULUS-2459] Executions List for a Granule

### PR DESCRIPTION
**Pull request by jaortega527**
**Summary:** Add Executions List for a Granule

Addresses [CUMULUS-2459: executions endpoint should be able to return a list of workflowNames](https://bugs.earthdata.nasa.gov/browse/CUMULUS-2459)

## Changes

* Added executions-list route and component
* Updated dev environment to support version of CUMULUS APIs pulling data from Postgres tables
* Updated database seed fixtures to support Postgres constraints
* Updated integration tests to accommodate seed fixture changes

## PR Checklist

- [x] Update CHANGELOG
- [x] Unit tests
- [x] Adhoc testing
- [x] Integration tests

### CODE REVIEW

**Comment by jjmccoy** [comment]
UI/UX: I will be reviewing for interface styling and interaction. I suggest another reviewer for code and to test against a large dataset.

**Comment by jjmccoy** [comment]
@jaortega527 Amazing work here. I just see a few things here.

Screen Shot 2021-08-06 at 11 11 14 AM

- An error is showing on the 'Executions List' page. I'm not sure if this is supposed to flag to the UI and what that means to the user.
- The 'Executions for ...' title's border doesn't go all the way in the container.
- The Executions List header is missing the description section. 
- The 'Total Executions' subtitle area is missing the grey fill-in. I would suggest something like this:
```
.heading__wrapper--border {
    margin-bottom: 1em;
    margin-top: 1.3em;
    padding: .8em 0em .3em 1em;
    background-color: #f3f3f3;
```
It's located at line 53 in _typography.scss

I'm suggesting the following for the header section for the scope of this ticket.
Screen Shot 2021-08-06 at 11 39 45 AM

**Comment by jaortega527** [comment]
@jjmccoy Great feedback. Making those changes now. I may have to reach out regarding the error. I'll let you know shortly.

**Comment by jaortega527** [comment]
Feedback addressed. I think the most recent change to seed-data may fix the error you were receiving as well.

File: app/src/js/utils/table-config/granules.js [hunk 1]
```diff
@@ -67,6 +67,16 @@ export const tableColumns = [
...
+  {
+    Header: 'Executions List',
+    accessor: 'granuleId',
+    Cell: ({ row: { original: { collectionId, granuleId } } }) => (// eslint-disable-line react/prop-types
+      <Link to={(location) => ({ pathname: `/executions/executions-list/${encodeURIComponent(collectionId)}/${encodeURIComponent(path.basename(granuleId))}` })}>link</Link>
+    ),
+    disableSortBy: true,
+    width: 90,
+    id: 'execution-list'
+  },
```
Comment (dopeters) [inline app/src/js/utils/table-config/granules.js:79]: Only real thing I'm noticing is that we have this column and we also still have the 'Execution' column. Should we change that to 'Latest Execution' maybe? Or get rid of it entirely?

Comment (jjmccoy) [reply app/src/js/utils/table-config/granules.js:79]: I suggest to get rid of it entirely.

Comment (jaortega527) [reply app/src/js/utils/table-config/granules.js:79]: Fair question. I'll get it removed.

Comment (jaortega527) [reply app/src/js/utils/table-config/granules.js:79]: @dopeters and @jjmccoy: The Execution column has been removed. I also adjusted the granules integration tests to accommodate that change.

**Comment by jjmccoy** [comment]
UI/UX will do another interface and interaction review since there have been significant changes but encourage another teammate to code and logic review as well.

**Comment by jjmccoy** [comment]
@jaortega527 I don't know why I'm still getting this error. Everything else is working fine as I go through the journey.

Screen Shot 2021-09-10 at 11 22 04 AM

File: package.json [hunk 2]
```diff
@@ -68,8 +68,8 @@
     "@babel/preset-react": "^7.10.4",
     "@babel/register": "^7.12.0",
     "@babel/runtime": "^7.12.0",
-    "@cumulus/api": "8.1.0",
     "@cumulus/aws-client": "8.1.0",
```
Comment (jennyhliu) [inline package.json:71]:
```suggestion
    "@cumulus/aws-client": "9.5.0",
```

**Review by dopeters — APPROVED** [review]
Looks good to me!

**Review by jjmccoy — COMMENTED** [review]
UI/UX: Interface & interactions LGTG!

**Pull request merged**